src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/02-Plugins.md

```diff
@@ -508,6 +508,29 @@ libraryDependencies ++=
         Seq.empty
 ```
 
+### Publishing a plugin
+
+Sbt plugins can be published as any other projects. 
+
+However, there is one caveat if you attempt to publish your plugin to a repository that follows the Maven scheme.
+For many years, Sbt plugins have utilized an artifact layout that is incompatible with Maven-based repositories, and rectifying this in a fully compatible manner has proven to be quite challenging.
+
+If your artifacts repository expect artifacts to be compliant with maven scheme and rejects artifacts that does no adhere to it you can:
+1. If you and consumers of your plugin use sbt 1.9.x
+   
+   Since sbt 1.9, it tries to publish any plugin with the new and legacy Maven style (for backward compatibility). The legacy Maven style is not fully compatible with Maven scheme.
```

Comment:
```suggestion
   Since sbt 1.9, it tries to publish any plugin with both the new and legacy Maven style (for backward compatibility). The legacy Maven style is not fully compatible with Maven scheme.
```